dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -145,24 +145,22 @@ private static String extractHTTPPrefix(String gitUrl, String reference, String
         }
 
         tool.setDescription(container.getDescription());
-        tool.setMetaVersion(container.getLastUpdated() != null ? container.getLastUpdated().toString() : new Date(0).toString());
+        tool.setMeta_version(container.getLastUpdated() != null ? container.getLastUpdated().toString() : new Date(0).toString());
```

Comment:
There should be a way to leave Java code in CamelCase while using snake case for the swagger and REST endpoints.